test/utils/image/OWNERS

```diff
@@ -4,12 +4,10 @@ reviewers:
   - justaugustus
   - luxas
   - mkumatag
-  - listx
   - dims
 approvers:
   - luxas
   - mkumatag
-  - listx
```

Comment:
FWIW we typically move folks to `emeritus_approvers`, it's not consumed by automation.  if you don't care or would rather not be there, feel free to `/hold cancel`